uint8 preset_number
float32 VBAT
uint16 STATE
uint32 FLAGS
float32 ABSOLUTE_HEADING
float32 ALTITUDE
float32 ALTITUDE_HOLD_SPEED
float32 GPS_ALT
float64 GPS_LAT
float32 GPS_LON
float32 GPS_HACC
float32 GPS_HDOP
uint8 DRONE_MODE
uint8 GPS_SATELLITES_FIX
uint8 GPS_NUMBER_OF_SATELLITES
int16 rc6_channel
int16 rc11_channel
uint16 current_wp
